tests/unit/OrgPerson.spec.ts

```diff
@@ -33,92 +33,92 @@ const cancelButtonSelector: string = '#btn-cancel'
 const formSelector: string = '.appoint-form'
 
 const validPersonData = {
-  'officer': {
-    'id': 0,
-    'firstName': 'Adam',
-    'lastName': 'Smith',
-    'middleName': 'D',
-    'orgName': '',
-    'partyType': 'Person',
-    'email': 'completing-party@example.com'
+  officer: {
+    id: '0',
+    firstName: 'Adam',
```

Comment:
BTW, this "object format" allows you to get auto-completion of the object's properties ;)